recipes-app/iot2050-firmware-update/files/iot2050-firmware-update.tmpl

```diff
@@ -121,12 +122,28 @@ class UpgradeError(Exception):
     def __str__(self):
         return self.erroinfo
 
+#
+# | Error code  | Description                              |
+# |-------------|------------------------------------------|
+# | 0           | Success                                  |
+# | 1           | Failed to parse the arguments            |
+# | 2           | User canceled                            |
+# | 3           | Failed to backup firmware                |
+# | 4           | Failed to rollback the original firmware |
+# | 5           | Internal error                           |
```

Comment:
The code is a good comment already IHMO. Better comment inline if some error code name is not verbose enough.

Reply:
Indeed, done.